it/test/controllers/NirmsQuestionControllerISpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import helpers.ItTestBase
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.{WSClient, WSRequest}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+
+class NirmsQuestionControllerISpec extends ItTestBase {
+
+  lazy val client: WSClient = app.injector.instanceOf[WSClient]
+
+  private val url = s"http://localhost:$port$appRouteContext/nirms-question"
+
+  "Nirms question controller" should {
+
+    "returns an error when auth fails" in {
```

Comment:
This describes that an error is returned but the logic checks that the status code is SEE_OTHER, can we modify this so that we can be certain an error is returned?

Reply:
have updated this now:)